tests/test_Sequence.cc

```diff
@@ -28,30 +28,49 @@
 using namespace task;
 using namespace std::literals;
 
-TEST_CASE("Sequence: Construct empty sequence", "[Sequence]")
-{
-    Sequence seq;
-    REQUIRE(seq.empty());
-    REQUIRE(seq.size() == 0);
-
-    static_assert(std::is_default_constructible<Sequence>::value,
-        "Sequence is_default_constructible");
-}
-
 TEST_CASE("Sequence: Constructor without descriptive name", "[Sequence]")
 {
     REQUIRE_THROWS_AS(Sequence(""), Error);
 }
 
 TEST_CASE("Sequence: Constructor with too long descriptive name", "[Sequence]")
 {
+    // label is to many characters -> throws Error
     REQUIRE_THROWS_AS(Sequence(std::string(Sequence::max_label_length + 1, 'c')), Error);
+    // empty label -> throws Error
+    REQUIRE_THROWS_AS(Sequence(""), Error);
```

Comment:
In the other functions you use curlies to create a `Sequence` object, very nice with the additional-blank style.
But here a function-alike with parens is used.

Maybe unify?

```suggestion
    REQUIRE_THROWS_AS(Sequence{ "" }, Error);
```

(Some few more places.)

Reply:
Agree: should be uniform. Fixed with curly brackets.